lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -2313,6 +2319,7 @@ def main():
         name=dict(type='str'),
         name_match=dict(type='str', choices=['first', 'last'], default='first'),
         uuid=dict(type='str'),
+        use_instance_uuid=dict(type='bool', default=False, required=False),
```

Comment:
If the parameter is not required, it doesn't need this explicit. Please do this for all occurances.
```suggestion
        use_instance_uuid=dict(type='bool', default=False),
```